README.md

```diff
@@ -34,3 +34,14 @@ Command line (using [sample.json](/sample.json)):
 ```sh
 cat sample.json | fx-crash-sig
 ```
+
+```sh
+python example.py
+```
+
+
+## Run tests
+
+```sh
+python setup.py test
+```
```

Comment:
I think this is how to run the tests. It ran the tests for me. If this isn't, then let me know and I can adjust this.